webapp/lib/model.dart

```diff
@@ -234,19 +35,27 @@ List<Map> messages = [
 ];
 
 class Configuration {
-  List<String> hasAllTags;
-  List<String> containsLastInTurnTags;
-  List<String> hasNoneTags;
+  Map<String, TagStyle> tags;
```

Comment:
I suggest we rename this to `availableTags` - otherwise it can be a bit difficult at first glance to understand how this is different to the other tags.

Reply:
I agree. Changed